cmd/commands/ceph.go

```diff
@@ -33,6 +35,10 @@ var CephCmd = &cobra.Command{
 	},
 	Run: func(cmd *cobra.Command, args []string) {
 		logging.Info("running 'ceph' command with args: %v", args)
-		exec.RunCommandInOperatorPod(cmd.Context(), clientSets, cmd.Use, args, operatorNamespace, cephClusterNamespace, false, true)
+		_, err := exec.RunCommandInOperatorPod(cmd.Context(), clientSets, cmd.Use, args, operatorNamespace, cephClusterNamespace, false)
+		if err != nil {
+			logging.Error(err)
+			os.Exit(1)
```

Comment:
```suggestion
			logging.Fatal(err)
```
`logging.Fatal` is combination of what you have

Reply:
Yeah, I should have read the implementation of `logging.Fatal` carefully.